proto-lens-protoc/src/Data/ProtoLens/Compiler/Definitions.hs

```diff
@@ -97,6 +110,20 @@ data MessageInfo n = MessageInfo
       -- unknown fields.
     } deriving Functor
 
+data ServiceInfo = ServiceInfo
+    { serviceName :: Text
+    , serviceMethods :: [MethodInfo]
+    }
+
+data MethodInfo = MethodInfo
+    { methodIdent :: Text
+    , methodPath :: Text
```

Comment:
I think this path structure ("/pkg.Foo/Bar") is specific to gRPC rather than being part of the official proto spec.  I suggest we just record the name of  the method ("Bar") and also the name of the service ("pkg.Foo") in a new field of `ServiceInfo`.)